docs/source/examples/beam.py

```diff
@@ -0,0 +1,46 @@
+from gpkit.shortcuts import *
+import numpy as np
+import matplotlib.pyplot as plt
+
+N = 100
+L = 5.
+dx = L/(N-1)
+ei = 1E4
+EI = Var("EI", ei)
+
+P = 100.
+p = Vec(N, "p", descr="Distributed load")
+p = p.sub(p, P*np.ones(N))
```

Comment:
Units of `P` are `N/m`. `P` shouldn't change if `N` changes, because `L` is fixed.